utils/bonjourmadame.py

```diff
@@ -2,18 +2,18 @@
 # -*- coding: utf-8 -*-
 """Bonjourmadame feed parser."""
 
-import requests
+import aiohttp
 from bs4 import BeautifulSoup
 
 
-def latest_madame():
+async def latest_madame():
     """Fetch last Bonjourmadame picture."""
     madames = "http://feeds2.feedburner.com/BonjourMadame"
-
-    res = requests.get(madames)
-    res.close()
-    soup = BeautifulSoup(res.text, 'lxml')
-
+    async with aiohttp.ClientSession() as session:
```

Comment:
Why not use 'get_soup_lxml()' from tools ?

Reply:
Yeah seems good, I didn't really think about it, I just changed `requests` to `aiohttp` 😅